Common/src/main/java/me/egg82/tfaplus/services/InternalAPI.java

```diff
@@ -95,36 +96,25 @@ public String registerHOTP(UUID uuid, long codeLength, long initialCounter, Jedi
 
         // SQL
         HOTPData result = null;
-        try {
-            if (sqlType == SQLType.MySQL) {
-                result = MySQL.updateHOTP(sql, storageConfigNode, uuid, codeLength, initialCounter, key).get();
-            } else if (sqlType == SQLType.SQLite) {
-                result = SQLite.updateHOTP(sql, storageConfigNode, uuid, codeLength, initialCounter, key).get();
-            }
-        } catch (ExecutionException ex) {
-            logger.error(ex.getMessage(), ex);
-        } catch (InterruptedException ex) {
-            logger.error(ex.getMessage(), ex);
-            Thread.currentThread().interrupt();
-        }
+        result = getHotpData(uuid, codeLength, initialCounter, key, result);
```

Comment:
Might as well change this to `HOTPData result = getHotpData(uuid, codeLength, initialCounter, key);`
Also, the extra `result` param is redundant.